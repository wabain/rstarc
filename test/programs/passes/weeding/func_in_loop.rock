while true is true
    InvalidBreak takes Dummy
        break
