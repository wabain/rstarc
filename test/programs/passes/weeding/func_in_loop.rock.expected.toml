[tests]
enabled = ["run"]

[run]
returncode = "1"
stderr = """
error: Unexpected loop control statement

3:          break
            ^^^^^
"""
